sphinx/developers/java-library.rst

```diff
@@ -10,6 +10,9 @@ the `OME Artifactory <http://artifacts.openmicroscopy.org/artifactory>`_.  The
 snippets for inclusion into your Gradle, Maven or Ivy project, which will
 enable the use of this repository.
 
+As of Bio-Formats 6.12.0 consumers will have to directly declare a dependency for 
```

Comment:
Only thought is that we might want to use markup to make this a bit more prominent as an action might be required.

Possibly the [versionchanged](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged) directive could be used here?